pkg/cloud/metadata/metadata.go

```diff
@@ -20,6 +20,7 @@ const (
 	ZoneID       MetadataKey = iota
 	InstanceID   MetadataKey = iota
 	InstanceType MetadataKey = iota
+	AliUID       MetadataKey = iota
```

Comment:
```suggestion
	AccountID    MetadataKey = iota
```
I remember that we use AccountID to reference main account UID across OpenAPIs. We may follow that. "Ali" does not make much sense since this repo is all about alibaba cloud.